### ISSUE TITLE
unethical!

### ISSUE THREAD

**Issue by onlypuppy7** [BRlMST0NE/Brim-s-FreeIP#1]
removes revenue from starving developers. harrison will be bankrupt! wizups will be homeless!

**Comment by enbyte** [comment]
I agree, this is extremely unethical. Please delete immediately and upload a video groveling to Wizups like you usually do instead.
